### DISCUSSION TITLE
Can the oh-my-posh debug command be used with PowerShell modules installation?

### DISCUSSION THREAD

**Discussion by ghost** [JanDeDobbeleer/oh-my-posh#506] [Troubleshoot] [answered]
I upgraded from oh-my-posh v2 and wanted to debug the performance as my oh-my-posh v3 installation was pretty slow. Doesn't look like the oh-my-posh command is available with a PowerShell module install. Is this intentional? If so, shouldn't it be mentioned in the docs?

**Comment by JanDeDobbeleer** [comment, marked as answer]
Not yet, but definitely a good idea! The executables are found in the module location inside the bin folder.

**Reply by JanDeDobbeleer** [reply]
@otaconjh v3.109.0 will have the `Write-PoshDebug` function for Powershell.
